src/index.js

```diff
@@ -1,7 +1,16 @@
 import 'regenerator-runtime/runtime'; //async/await with Parcel
 import { App } from './app/App';
 
-const BASE_API_URL = process.env.API_BASE_URL || 'https://google.com';
+window.addEventListener('click', (e) => {
+    if (e.target instanceof HTMLAnchorElement) {
```

Comment:
Can't se an option to pass a parameter to the route, like asset id which we will need